subsys/caf/modules/power_manager.c

```diff
@@ -307,7 +307,9 @@ static bool event_handler(const struct event_header *eh)
 	return false;
 }
 EVENT_LISTENER(MODULE, event_handler);
-EVENT_SUBSCRIBE(MODULE, keep_alive_event);
+#if IS_ENABLED(CONFIG_CAF_KEEP_ALIVE_EVENTS)
```

Comment:
CONFIG_CAF_KEEP_ALIVE_EVENTS is actually selected by the CONFIG_CAF_POWER_MANAGER. Do we need the #if here ?

Reply:
Not all the applications uses keep_alive_event - I would disable it by default.

Reply:
ok